### ISSUE TITLE
Plans to extend beyond alpine?

### ISSUE THREAD

**Issue by pdxjohnny** [chainguard-dev/apko#42]

**Comment by dekarl** [comment]
> `apko` already supports all of the APK ecosystem distributions such as Alpine, Adelie, Chimera and OpenWrt, but there is no plan to extend this tool beyond the APK ecosystem.

Mind the bump. I tried to build an OpenWRT image, but due to OpenWRT using the new APKv3 formats I failed. Any plans / timeline to add support for APKv3?

It appears OpenWRT are the only ones already using the new file formats.
https://7ji.github.io/designdoc/2026/03/03/into-Alpine-APK-v3-format-the-binary-perspective.html
https://wiki.alpinelinux.org/wiki/Alpine_Package_Keeper

Now that someone has collected the format and written an independent implementation to extract the packages could be a good time to start looking into it and a go implementation.

Seeing the variants of "small Linux" working on shared tools to use the same concepts and tools for minimal Linux is inspiring. Looking forward to build OCI images to run networking software on clusters and application packages to run on switches. All from the same builds.
